tactics/license-plate-readers.html

```diff
@@ -43,35 +43,34 @@
     <div class="page-img"><img src="../img/Car_4.gif" alt="" style=""></div>
     <div class="page-txt">
         <h1 class="home-head">License Plate Readers</h1>
-        
-    <p>Automatic License Plate Recognition, or ALPR, take images of vehicles in public spaces, analyze the text on their license plates, and record the timestamp and position of the car (based on GPS coordinates of the camera).</p>
+
+    <p>Automatic License Plate Recognition, or ALPR, take images of vehicles in public spaces, analyze the text on their license plates, and record the timestamp and position of the car's license (based on GPS coordinates of the camera.)</p>
```

Comment:
"timestamp and position of their licenses" since we said "vehicles" and "license plates" in plural.